### ISSUE TITLE
-jobconf option is deprecated

### ISSUE THREAD

**Issue by nschiefer** [klbostee/dumbo#45]
Whenever a jumbo job is run, this warning appears:

`11/07/07 13:23:25 WARN streaming.StreamJob: -jobconf option is deprecated, please use -D instead.`

**Comment by klbostee** [comment]
Yup, we were already aware of this but it's definitely a good idea to create a github issue for it. Patches are welcome.. :)

**Comment by supreet11agrawal** [comment]
@nschiefer @klbostee This is my first time on GitHub and I seriously want to contribute by fixing bugs. Can you please help me with that?
